### PR TITLE
Json logging

### PR DESCRIPTION
**Pull request by whereismyjetpack**
- Removes the taggedlogger (since this info is in lograge output) 
- adds the json formatter so that logs in splunk are easier to read, parse, and ... 'splunk'

Here's a screenshot of the logs before, and after in splunk.

Before:
Screen Shot 2021-08-19 at 10 47 40 AM
After:
Screen Shot 2021-08-19 at 10 46 59 AM

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@
   # config.active_support.deprecation = :notify
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
```
Comment (banukutlu) [inline config/environments/production.rb:72]: @whereismyjetpack  is this comment relevant?

**Review by lauterman — APPROVED** [review]

File: config/environments/production.rb (hunk 1 above)
Comment (whereismyjetpack) [reply config/environments/production.rb]: it is not! mended.

Comment (banukutlu) [reply config/environments/production.rb]: thanks!

**Review by banukutlu — APPROVED** [review]
lgtm 👌🏻  thanks Dann, formatted logs are super helpful!

**Pull request merged**